### PR TITLE
Update paths and types in order to run train_distill.

### PR DESCRIPTION
**Pull request by entrpn**
# Description

Update paths and types in order to run train_distill.

# Tests

Ran existing tests in two files, `tests.unit.train_distill_test` and `tests.unit.distillation_data_processing_test`

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code. For an optional AI review, add the `gemini-review` label.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run end-to-end tests tests and provided workload links above if applicable.
- [x] I have made or will make corresponding changes to the doc if needed, including adding new documentation pages to the relevant Table of Contents (toctree directive) as explained in [our documentation](https://maxtext.readthedocs.io/en/latest/development.html#adding-new-documentation-files).

### CODE REVIEW

File: src/MaxText/configs/types.py [hunk 1]
```diff
@@ -1019,6 +1018,7 @@ class Distillation(BaseModel):
   # --- Loss Params ---
   distill_alpha: float = Field(0.5, description="Weight for the distillation loss component.")
   distill_temperature: float = Field(1.0, description="Temperature for distillation softening.")
+  cosine_learning_rate_final_fraction: float = Field(0.1, description="Cosine learning rate final fraction.")
```
Comment (gagika) [inline src/MaxText/configs/types.py]: could you just reuse existing `learning_rate_final_fraction`

https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/configs/types.py#L1066

I think this PR: https://github.com/AI-Hypercomputer/maxtext/pull/2883 didn't update the distillation script

**Review by RissyRan — COMMENTED** [review]
@gemini-cli /review

File: src/MaxText/configs/types.py [hunk 2]
```diff
@@ -1018,6 +1018,7 @@ class Distillation(BaseModel):
...
```
[the remark this answers is not in the document]
Comment (entrpn) [reply src/MaxText/configs/types.py]: using learning_rate_final_fraction instead of adding a new type since it works with cosine.

File: src/MaxText/configs/types.py (hunk 1 above)
Comment (entrpn) [reply src/MaxText/configs/types.py]: done
